themes/default/layouts/partials/docs/right-nav.html

```diff
@@ -1,12 +1,15 @@
 {{ with .File }}
     {{ if not $.Page.Params.no_on_this_page }}
-        <div class="on-this-page">
-            <h4 class="no-anchor">
-                <a href="#">On This Page</a>
-            </h4>
-            <ul class="text-sm">
-                {{/* This will be populated dynamically with all H2s and H3s on the page; see assets/js/main.js */}}
-            </ul>
+        <div class="on-this-page table-of-contents">
```

Comment:
do we need this `on-this-page` class anymore? it looks like we removed/renamed it. if not ignore me.

Reply:
@sean1588 this is actually used in the ts file 🙃 
ive cleaned that up though in my incoming commit